docs/Backtracing.md

```diff
@@ -0,0 +1,166 @@
+| timeout          | 30s      | Time to wait for interaction when a crash occurs. Setting this to `none` or `0s` will disable interaction. |
+| unwind           | auto     | Specifies which unwind algorithm to use. `auto` means to choose appropriately for the platform. Other options are `fast`, which does a naïve stack walk; and `precise`, which uses exception handling data to perform an unwind. |
+| preset           | auto     | Specifies which set of preset formatting options to use. Options are `friendly`, `medium` or `full`. `auto` means to use `friendly` if interactive, and `full` otherwise. |
+| sanitize         | preset   | If `yes`, we will try to process paths to remove PII. Exact behaviour is platform dependent. |
+| threads          | preset   | Options are `all` to show backtraces for every thread, or `crashed` to show only the crashing thread. |
+| registers        | preset   | Options are `none`, `all` or `crashed`.                                                |
+| images           | preset   | Options are `none`, `all`, or `mentioned`, which only displays images mentioned in a backtrace.
+| limit            | 64       | Limits the length of the captured backtrace. See below for a discussion of its behaviour. Can be set to `none` to mean no limit. |
+| top              | 16       | Specify a minimum number of frames to capture from the top of the stack. See below for more. |
+| cache            | yes      | Set to `no` to disable symbol caching. This only has effect on platforms that have a symbol cache that can be controlled by the runtime. |
+| swift-backtrace  |          | If specified, gives the full path to the swift-backtrace binary to use for crashes. Otherwise, Swift will locate the binary relative to the runtime library, or using `SWIFT_ROOT`. |
+
+[^*]: On macOS, this defaults to `tty` rather than `yes`.
+
+## Backtrace limits
+
+The limit settings are provided both to prevent runaway backtraces and
+to allow for a sensible backtrace to be produced even when a function
+has blown the stack through excessive recursion.
+
+Typically in the latter case you want to capture some frames at the top
+of the stack so that you can see how the recursion was entered, and the
+frames at the bottom of the stack where the actual fault occurred.
+
+1.  There are `limit` or fewer frames. In this case we will display all
+    the frames in the backtrace. Note that this \_[includes]() the case
+    where there are exactly `limit` frames.
+2.  There are more than `limit` frames.
+    1.  `top` is `0`. We will display the first `limit - 1` frames
+        followed by `...` to indicate that more frames exist.
+    2.  `top` is less than `limit - 1`. We will display
+        `limit - 1 - top` frames from the bottom of the stack, then a
+        `...`, then `top` frames from the top of the stack.
+    3.  `top` is greater or equal to `limit - 1`. We will display `...`,
+        followed by `limit - 1` frames from the top of the stack.
```

Comment:
How about changing all these bullet introducers to `1.` and letting the renderer take care of enumerating them instead?